### PR TITLE
LGA-954 Welsh Part 2

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
## What does this pull request do?

New phrases added

## Any other changes that would benefit highlighting?

Some phrases were deleted as part of the automatic update process, these have been restored.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: cla_public/translations/cy/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -1388,44 +1421,37 @@ msgstr "Cyngor Cyfreithiol Sifil"
...
 #, python-format
 msgid ""
 "\n"
-"          If you’re in immediate danger please call Civil Legal Advice on <span class=\"tel\">0345 345 4 345</span>.\n"
-"          %(call_charges_link)s.\n"
-"        "
+"            If you’re in immediate danger please call Civil Legal Advice on <strong class=\"laa-telephone\">0345 345 4 345</strong>.\n"
+"            %(call_charges_link)s.\n"
+"          "
 msgstr ""
```
Comment (said-moj) [inline cla_public/translations/cy/LC_MESSAGES/messages.po]: Was this translation removed on purpose

File: cla_public/translations/cy/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -3349,10 +3310,117 @@ msgid ""
...
+
+#: cla_public/templates/checker/result/ineligible.html:57
+#, python-format
+msgid ""
+"\n"
+"        If you want to make an application to court about a family matter you need to first of all\n"
+"        see if family mediation will help. Use the %(family_mediation_link)s directory to find a mediator\n"
+"        and make an appointment for a Mediation Information and Assessment Meeting (MIAM).\n"
+"      "
 msgstr ""
```
Comment (said-moj) [inline cla_public/translations/cy/LC_MESSAGES/messages.po]: Another one of missing translation

File: cla_public/translations/cy/LC_MESSAGES/messages.po (hunk 1 above)
Comment (MalcolmVonMoJ) [reply cla_public/translations/cy/LC_MESSAGES/messages.po]: fixed

File: cla_public/translations/cy/LC_MESSAGES/messages.po (hunk 2 above)
Comment (MalcolmVonMoJ) [reply cla_public/translations/cy/LC_MESSAGES/messages.po]: This translation that was deleted is not the translation of this phrase.  One for round 3 I think.

**Pull request merged**